crates/biome_configuration/src/javascript/formatter.rs

```diff
@@ -103,6 +103,11 @@ pub struct JsFormatterConfiguration {
     #[bpaf(long("bracket-spacing"), argument("true|false"))]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub bracket_spacing: Option<BracketSpacing>,
+
+    /// Whether to enforce collapsing object literals when possible. Defaults to preserve.
+    #[bpaf(long("object-wrap"), argument("preserve|collapse"))]
```

Comment:
This argument should be prefixed with `javascript`, like the args are since this is javascript specific. eg `--javascript-object-wrap`

```suggestion
    #[bpaf(long("javascript-object-wrap"), argument("preserve|collapse"))]
```

Reply:
Fixed in [f220b66](https://github.com/biomejs/biome/pull/5068/commits/f220b665260ed4e4d595730d272380fafda6f33b).